c2corg_api/models/document.py

```diff
@@ -53,15 +67,101 @@ class (e.g. waypoint, route, ...) a separate table will be created, which
     # TODO constraint that there is at least one locale
     locales = relationship('DocumentLocale')
 
-    _ATTRIBUTES = ['document_id', 'protected', 'redirects_to', 'quality']
+    __mapper_args__ = {
+            'version_id_col': _DocumentMixin.version,
+            'version_id_generator': lambda version: uuid.uuid4().hex
```

Comment:
Shouldn't we use a smaller number?
An uuid4 is billions of billions of billions of possible values: 16^32 = 340282366920938463463374607431768211456.